### PR TITLE
Better support for Subscriptions in the IPN admin

### PR DESCRIPTION
**Pull request by tockrock**
This pull request expose the subscription related elements that are currently not displayed.

### CODE REVIEW

**Comment by spookylukey** [comment]
Thanks for this, I'm unavailable for a few weeks but will look at it then, and your other PR. Please ping me if I don't.

**Comment by tockrock** [comment]
No problem. Thanks for maintaining  this project!

**Comment by tockrock** [comment]
Hey, just wanted to ping you to check if you're available.

File: paypal/standard/models.py [hunk 1]
```diff
@@ -240,6 +240,8 @@ class Meta:
     def __unicode__(self):
         if self.is_transaction():
             return self.format % ("Transaction", self.txn_id)
+        elif self.subscr_id:
```
Comment (spookylukey) [inline paypal/standard/models.py]: This should be `is_subscription()`, which should wrap the logic just like `is_transaction()`

**Comment by spookylukey** [comment]
Other than my comment above, looks good.

**Comment by tockrock** [comment]
Good point, pushed another commit with your suggestion.

**Comment by spookylukey** [comment]
Thanks!

**Pull request merged**